easybuild/easyconfigs/a/ADIOS2/ADIOS2-2.10.2-gompi-2024a.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = "CMakeMake"
+
+name = "ADIOS2"
+version = "2.10.2"
+
+homepage = 'https://github.com/ornladios/ADIOS2/'
+description = """This is ADIOS2: The Adaptable Input/Output (I/O) System.
+ADIOS2 is developed as part of the United States Department of Energy's Exascale Computing
+Project. It is a framework for scientific data I/O to publish and subscribe to data when
+and where required.
+ADIOS2 transports data as groups of self-describing variables and attributes across different
+media types (such as files, wide-area-networks, and remote direct memory access) using a
+common application programming interface for all transport modes. ADIOS2 can be used on
+supercomputers, cloud systems, and personal computers. """
+
+toolchain = {'name': 'gompi', 'version': '2024a'}
+
+source_urls = ['https://github.com/ornladios/ADIOS2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['14cf0bcd94772194bce0f2c0e74dba187965d1cffd12d45f801c32929158579e']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Python', '3.12.3'),
+]
+
+dependencies = [
+    ('HDF5', '1.14.5'),
+    ('c-blosc', '1.21.6'),
+    ('libfabric', '1.21.0'),
```

Comment:
hm i'm very suspicious about having this dependency in here. I filter this out in my openmpis and i can't imagine it being useful here for most people though.

There are several other dependencies for ADIOS that seem more relevant, like UCX